<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddTable1</class>
 <widget class="QDialog" name="DialogAddTable1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>175</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>WenQuanYi Micro Hei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>增加项目</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>识别1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>说明 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem0">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btRecordStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btRecordFinish">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>录音完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAddConfirm">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定写入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btQuit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
